.github/workflows/publish.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish
+
+on:
+  pull_request:
+    branches: [ master, dev ]
```

Comment:
```suggestion
    branches: [ master ]
```

Reply:
can't hurt, can it?

Reply:
> can't hurt, can it?

you mean, "it can't hurt to keep it there"? i guess not 🤷 

I'll assume you use it in a personal workflow or something, or aspire to propose a `dev` branch. regardless, if you'd prefer not to remove, it's the sort of thing I try not to care about splitting hairs over :)